ws-nextjs-app/pages/[service]/live/[id]/Post/index.test.tsx

```diff
@@ -151,6 +156,21 @@ describe('Post', () => {
       ).toBeInTheDocument();
     });
 
+    it('should render the new media player in a post containing audio', async () => {
+      const { container } = await act(async () => {
+        return render(<Post post={audioSamplePost} />, {
+          id: 'c7p765ynk9qt',
+          service: 'pidgin',
+          pageType: LIVE_PAGE,
+          pathname: '/pidgin/live/c7p765ynk9qt',
+        });
+      });
+
+      expect(
+        container.querySelector('[data-e2e="media-loader"]'),
```

Comment:
Heya noticed this is still failing. Is there a component with data-e2e "media-loader"? I can get this to pass if I change this to "media-loader__container".

Reply:
Sorry yes good catch ! I had changed this preemptively when i first removed the placeholder for the audio and clearly didnt double check it correctly woops